lib/Server.js

```diff
@@ -1133,13 +1133,15 @@ class Server {
 
           // cert is more than 30 days old, kill it with fire
           if ((now - Number(certificateStat.ctime)) / certificateTtl > 30) {
-            const del = require("del");
+            const { promisify } = require("util");
+            const rimraf = require("rimraf");
+            const del = promisify(rimraf);
 
             this.logger.info(
               "SSL certificate is more than 30 days old. Removing..."
             );
 
-            await del([certificatePath], { force: true });
```

Comment:
`certificatePath` is not a glob, it's the result of calling `const certificatePath = path.join(certificateDir, "server.pem")` on line 1119. 
Also line 1123 uses `fs.stat` on `certificatePath` to check for existence. 
`{force: true}` bypasses a check added by `del`, so it's not needed for rimraf.